lib/typescript/render/providers/chatplugin/ChatPluginRender.tsx

```diff
@@ -60,7 +58,7 @@ function render(content: ContentUnion, props: RenderPropsUnion) {
   }
 }
 
-function mapContent(message: Message): ContentUnion {
+function mapContent(message): ContentUnion {
```

Comment:
👀

Reply:
Yes this was not accurate: We use the renderer for templates and suggested replies also, which are **not** messages. The actual interface is:
```typescript
interface message {
  content: any;
  fromContact?: bool
}
```

Reply:
Therefore I removed the explicit dependency on message cc @airyhq/codeowners-frontend